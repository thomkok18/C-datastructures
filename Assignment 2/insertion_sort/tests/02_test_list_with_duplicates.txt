10
10
11
8
1
2
3
8
