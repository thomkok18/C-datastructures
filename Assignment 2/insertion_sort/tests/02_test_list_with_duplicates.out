1
2
3
8
8
10
10
11
